pkg/operation/botanist/component/kubeapiserver/kube_apiserver.go

```diff
@@ -165,6 +165,9 @@ type Values struct {
 	OIDC *gardencorev1beta1.OIDCConfig
 	// Requests contains configuration for the kube-apiserver requests.
 	Requests *gardencorev1beta1.KubeAPIServerRequests
+
+	// CORSAllowedOrigins contans a list of List of allowed origins for CORS
+	CORSAllowedOrigins []string
```

Comment:
Could you adapt the unit test to configure this field and ensure that the corresponding flag is set correctly?